src/styles-breakpoints.scss

```diff
@@ -0,0 +1,41 @@
+// setting breakpoint sizes here
+$xs: 0px;
```

Comment:
map should be more easier to read 
https://css-tricks.com/snippets/sass/mixin-manage-breakpoints/

Reply:
I see

Reply:
แต่ถ้าใช้ท่านั้นจะทำ only ยากกว่านะครับ